.storybook/webpack.config.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
```

Comment:
I'm wondering why linter doesn't complain. It might be because of the leading `.` in the name of the folder.

Reply:
Yes, confirmed. I changed the folder name to `storybook` to mitigate this issue.